### ISSUE TITLE
[DEV placeholder] Cypress End to End Test 

### ISSUE THREAD

**Issue by ldelacosta** [department-of-veterans-affairs/va.gov-team#34281]
Background: The Testing Tools Team are sharing their latest recommendations for writing Cypress end to end tests. 
They are adding an ESLint rule that will require that at least one accessibility check is included in each Cypress test. If a test is missing an accessibility check, the rule will show a warning with a link to our documentation on automated accessibility testing. We ask that teams audit their existing tests to ensure that all Cypress tests include accessibility checks. The rule will be added to `vets-website` on 12/15. 

In order for this rule to work locally, we will need to run `yarn install --check-files` in vets-website after pulling down the commit that includes this rule. The [Cypress documentation](https://depo-platform-documentation.scrollhelp.site/developer-docs/A11y-Testing.1935409178.html) includes a section on accessibility testing.
